src/cli/plugin/pluginDownloader.js

```diff
@@ -90,6 +96,14 @@ module.exports = function (settings, logger) {
     });
   }
 
+  function getArchiveTypeFromFilename(filePath) {
+    if (/.zip$/.test(filePath)) {
+      return '.zip';
+    }
+    if (/.tar.gz$/.test(filePath)) {
```

Comment:
You're not escaping the `.`'s, and I'm pretty sure that's a typo. Unless you really are trying to match `filenamextarxgz`